.github/workflows/build-images.yaml

```diff
@@ -115,15 +115,12 @@ jobs:
           sudo apt-get install -y qemu-user-static
       - name: Run make bundle (main)
         if: ${{ github.ref_name == env.MAIN_BRANCH_NAME }}
-        run: make bundle REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }}
+        run: make bundle REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }} IMAGE_TAG=${{ github.sha }}
       - name: Run make bundle (release)
         if: ${{ github.ref_name != env.MAIN_BRANCH_NAME }}
         run: make bundle REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }} VERSION=${TAG_NAME/v/} AUTHORINO_VERSION=${{ github.event.inputs.authorinoVersion }}
       - name: Git diff
         run: git diff
-      - name: Verify manifests and bundle (main)
```

Comment:
Then this step could be added back

Reply:
Regardless, I would not remove this check. It ensures (i) that the bundle generation is idempotent, and (ii) that we did not forget to commit any important change in the copy of the manifests that is pushed to remote.

Hypothetically – Ignoring what @didierofrivia [said](https://github.com/Kuadrant/authorino-operator/pull/132/files#r1289908207) for a second, if we were to include the SHA in the bundle manifests (like proposed here), then, to prevent from failing at this step (since that SHA would not match what I imagine would have been committed and pushed to remote, i.e. `latest`), the verification of the bundle must be enhanced.

Reply:
> Regardless, I would not remove this check. It ensures (i) that the bundle generation is idempotent, and (ii) that we did not forget to commit any important change in the copy of the manifests that is pushed to remote.

Just to note the [verify-manifests](https://github.com/Kuadrant/authorino-operator/blob/main/.github/workflows/test.yaml#L11) check that runs on PR's is unchanged - the check we removed above only runs when commits are merged to main (which should have already passed the verify-manifests check) or on `workflow_dispatch`. We could add to the test workflow the `make verify-bundle` command so that PR's can't be missing the changes in the manifests/bundles, but this would still mean that we are making the assumption that the main branch is always in a valid state as a result of this.